File: rb/test/exists_test.rb
# Elonmuskapi SDK exists test

require "minitest/autorun"
require_relative "../Elonmuskapi_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ElonmuskapiSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- Elonmuskapi SDK exists test

local sdk = require("elonmuskapi_sdk")

describe("ElonmuskapiSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
